### PR TITLE
feat: #30 #111 implements TV with nextjs

### PR DESCRIPTION
**Pull request by geun**
Simple implements with nextjs 9.x

`libraryPath` should use `static` folder 

Fixes: #30, #111

### CODE REVIEW

File: nextjs-javascript/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# TradingView Charting Library and Nextjs Integration Example (JavaScript)
+
+## How to start
+
+1. Install dependencies `npm install`.
+1. Copy `charting_library` folder from https://github.com/tradingview/charting_library/ to `/static` folders. The earliest supported version of the Charting Library is 1.12. If you get 404 then you need to [request an access to this repository](https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/).
```
Comment (timocov) [inline nextjs-javascript/README.md]:
```suggestion
1. Copy `charting_library` folder from https://github.com/tradingview/charting_library/ to `/static` folders. The earliest supported version of the Charting Library is 1.14. If you get 404 then you need to [request an access to this repository](https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/).
```

File: nextjs-javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+1. Copy `datafeeds` folder from https://github.com/tradingview/charting_library/ to `/static`.
+1. Run `npm start`. It will build the project and open a default browser with the Charting Library.
```
Comment (timocov) [inline nextjs-javascript/README.md]: I just tried to run in on clean env and it says that

```
$ npm run start

> charting-library-next-js-example@1.0.0 start E:\git\github\charting-library-examples\nextjs-javascript
> next start

Error: Could not find a valid build in the 'E:\git\github\charting-library-examples\nextjs-javascript\.next' directory! Try building your app with 'next build' before starting the server.
```

It looks like we miss `npm run build` before `npm run start`?

Comment (geun) [reply nextjs-javascript/README.md]: @timocov yes. next js should use after build for production. I will add it about dev mode and build for production. thanks

**Comment by geun** [comment]
@timocov  I update Reame.md for dev and production

**Comment by timocov** [comment]
@geun thank you a lot for this sample!

**Pull request merged**